research/gaia/gremlin/gremlin_core/src/process/traversal/step/flat_map/mod.rs

```diff
@@ -46,7 +46,13 @@ impl FlatMapFuncGen for pb::GremlinStep {
                     vertex_step.gen_flat_map()
                 }
                 pb::gremlin_step::Step::PropertiesStep(properties_step) => {
-                    Ok(Box::new(PropertiesStep { props: properties_step.properties.clone(), tags }))
+                    // TODO: should pass prop_name or prop_id
+                    let prop_keys = properties_step
```

Comment:
Better use the same name as the attribute's name in `PropertiesStep`. Either do: `let props = ...` or define `PropertiesStep { prop_keys: PropKey, ... }`

Reply:
Done